blueocean-jira/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
huh - I wonder if it would be possible to get this upstreamed in the jira plugin itself? it would need to depend on blue ocean rest right?

Reply:
@michaelneale yeah, that's the right thing to do. same with some of the git and github stuff, but we can tombstone these small plugins when we get around to properly splitting this stuff up. I was asked this very question at JW and think the right answer is to implement a transitional plugin until we can get the functionality merged to the upstream one, they can use an optional dependency on blue ocean.

Reply:
@kzantow ok - fair enough. Wil need to open the permission PR for a new plugin (and I will get chewed out a bit). 

Also makes it easier to get upstream when the blue ocean extension points are published and stable.